README.md

```diff
@@ -1,6 +1,53 @@
 # terraform-aws-vpc
 
-Terraform module that defines a VPC with public/private subnets across multiple AZs with Internet Gateways
+Terraform module that defines a VPC with Internet Gateway
```

Comment:
Mention that the user should us `terraform-aws-dynamic-subnets` or `terraform-aws-named-subnets` to add subnets to the VPC.